### PR TITLE
DOP-4499 Reducing file size for Noto Sans SC and KR fonts

### PR DESCRIPTION
**Pull request by caesarbell**
### Stories/Links:

DOP-4499

### Current Behavior:

[Simplified Chinese Staging Site from DOP-4450](https://docs-mongodb-org-stg.s3.us-east-2.amazonaws.com/zh-cn/master/docs/caesarbell/DOP-4450/index.html)
[Korean Staging Site from DOP-4450](https://docs-mongodb-org-stg.s3.us-east-2.amazonaws.com/ko-kr/master/docs/caesarbell/DOP-4450/index.html)

### Staging Links:

[Simplified Chinese Staging Site](https://docs-mongodb-org-stg.s3.us-east-2.amazonaws.com/zh-cn/master/cloud-docs/caesarbell/DOP-4499/index.html)
[Korean Staging Site](https://docs-mongodb-org-stg.s3.us-east-2.amazonaws.com/ko-kr/master/cloud-docs/caesarbell/DOP-4499/index.html)

### Notes:

The purpose of this PR is to provide a solution to reduce the bundle size for the Noto Sans font. From the ticket `DOP-4450`, we introduced the fonts in gatsby-ssr.js  and preloaded it, to then use it in the `mongodb-docs.css` via font-face. 

The concern at that time was the bundle size. Those new fonts are considerably larger in size than other fonts being loaded. 

### README updates

- - [ ] This PR introduces changes that should be reflected in the README, and I have made those updates.
- - [x] This PR does not introduce changes that should be reflected in the README

### CODE REVIEW

File: gatsby-ssr.js [hunk 1]
```diff
@@ -25,8 +23,10 @@ export const onRenderBody = ({ setHeadComponents }) => {
         __html: `!function(e,t,r,n){if(!e[n]){for(var a=e[n]=[],i=["survey","reset","config","init","set","get","event","identify","track","page","screen","group","alias"],s=0;s<i.length;s++){var c=i[s];a[c]=a[c]||function(e){return function(){var t=Array.prototype.slice.call(arguments);a.push([e,t])}}(c)}a.SNIPPET_VERSION="1.0.1";var o=t.createElement("script");o.type="text/javascript",o.async=!0,o.src="https://d2yyd1h5u9mauk.cloudfront.net/integrations/web/v1/library/"+r+"/"+n+".js";var l=t.getElementsByTagName("script")[0];l.parentNode.insertBefore(o,l)}}(window,document,"Dk30CC86ba0nATlK","delighted");`,
       }}
     />,
-    <link rel="preload" href={NotoSansKR} as="font" type="font/woff2" crossOrigin="anonymous" key="notoSansKR" />,
-    <link rel="preload" href={NotoSansSC} as="font" type="font/woff2" crossOrigin="anonymous" key="notoSansSC" />,
+    <link rel="preconnect" href="https://fonts.googleapis.com" />,
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />,
```
Comment (seungpark) [inline gatsby-ssr.js:27]:
```suggestion
    <link rel="preconnect" href="https://fonts.gstatic.com" crossOrigin />,
```

minor warning here for `Warning: Invalid DOM property `crossorigin`. Did you mean `crossOrigin`?`

**Review by seungpark — APPROVED** [review]
i think ideally we would want it in our fonts to really preload before page load. but then again, we need to load the components and read the url path so this reduction is a huge plus 👍

File: gatsby-ssr.js (hunk 1 above)
Comment (caesarbell) [reply gatsby-ssr.js:27]: @seungpark good catch, it is camelCase and takes a string. i.e. `crossOrigin="anonymous"`

**Review by rayangler — APPROVED** [review]
Nice I was about to leave a comment about the keys but saw that you fixed it at the same time 😎. Can we please update the PR title to include context?

**Comment by caesarbell** [comment]
> Nice I was about to leave a comment about the keys but saw that you fixed it at the same time 😎. Can we please update the PR title to include context?

I truly have to remind myself to provide titles with more context. I am used to just having the ticket number as the title.

**Pull request merged**